src/core/renderers/webgl/WebGLRenderer.js

```diff
@@ -440,18 +440,22 @@ export default class WebGLRenderer extends SystemRenderer
      * Changes the current shader to the one given in parameter
      *
      * @param {PIXI.Shader} shader - the new shader
+     * @param {boolean} [autoProject=true] - Whether automatically set the projection matrix
      * @return {PIXI.WebGLRenderer} Returns itself.
      */
-    bindShader(shader)
+    bindShader(shader, autoProject)
```

Comment:
'leave a comment explaining something' is very difficulty for  me , you know my english is very bad.   tears...